src/components/NcAppSidebar/NcAppSidebarTabsButton.vue

```diff
@@ -0,0 +1,111 @@
+<!--
+  - SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import type NcAppSidebarTab from '../NcAppSidebarTab/NcAppSidebarTab.vue'
+
+import NcVNodes from '../NcVNodes/NcVNodes.vue'
+
+const selected = defineModel<boolean>('selected', { required: true })
+
+defineProps<{
+	/**
+	 * The sidebar tab this button controls
+	 */
+	tab: InstanceType<typeof NcAppSidebarTab>
+}>()
+</script>
+
+<template>
+	<button
+		class="button-vue"
+		:class="[$style.sidebarTabsButton, {
+			[$style.sidebarTabsButton_selected]: selected,
+		}]"
+		role="tab"
+		:aria-selected="selected"
+		:tabindex="selected ? 0 : -1"
+		@click="selected = true">
+		<span :class="$style.sidebarTabsButton__icon">
+			<NcVNodes :vnodes="tab.renderIcon()">
+				<span :class="[$style.sidebarTabsButton__legacyIcon, tab.icon]" />
+			</NcVNodes>
+		</span>
+		<span :class="$style.sidebarTabsButton__name">
+			{{ tab.name }}
+		</span>
+	</button>
+</template>
+
+<style module lang="scss">
+.sidebarTabsButton {
+	border: none;
+	border-bottom: var(--default-grid-baseline) solid transparent !important;
+	border-radius: var(--border-radius-small);
+	background-color: var(--color-main-background);
+	color: var(--color-main-text);
```

Comment:
Setting `font-size` is missing.

```suggestion
	color: var(--color-main-text);
	font-size: var(--default-font-size);
```

Reply:
For me it had `inherit` set and then it was set implicitly to 15 :eyes: